synapse/storage/data_stores/main/e2e_room_keys.py

```diff
@@ -288,21 +370,31 @@ def _create_e2e_room_keys_version_txn(txn):
         )
 
     @trace
-    def update_e2e_room_keys_version(self, user_id, version, info):
+    def update_e2e_room_keys_version(
+        self, user_id, version, info=None, version_etag=None
```

Comment:
if `info` is now allowed to be None, please update the type in docstring accordingly. what happens if it's not set?